### DISCUSSION TITLE
Breaking Changes for May 2023

### DISCUSSION THREAD

**Discussion by nateswart** [etsy/open-api#982] [Announcements]
On May 31, 2023 the following breaking changes will be released. Please have your integrations updated before this date to ensure no disruptions in service to our users.

- We recently introduced the boolean filter `was_canceled` to the `getShopReceipts` endpoint and defaulted the value `false`. We will be updating this default now to be `null` which will return all receipts, including those will a canceled status.
- We will be adding `Fully Refunded` and `Partially Refunded` statuses to the non-canceled orders for better visibility on `getShopReceipts`.
- We will limit the length of alt_text text in `uploadListingImage` to 250 characters to match the UI entry within the Etsy website.

**Comment by jasonpincin** [comment]
This release is out. See https://github.com/etsy/open-api/releases/tag/3.0.0-general-release-2023-05-31
